src/main/java/appeng/block/AEBaseBlock.java

```diff
@@ -150,8 +121,8 @@ public String toString()
 
 	public void registerNoIcons()
 	{
-		BlockRenderInfo info = this.getRendererInstance();
-		FlippableIcon i = new FlippableIcon( new MissingIcon( this ) );
+		final BlockRenderInfo info = this.getRendererInstance();
```

Comment:
Welcome to the magic of java generics and reflection. :grinning: 

The best option is probably to refactor `AEBaseBlock.getRenderer()` to no longer return a `Class<? extends BaseBlockRender>` and instead directly construct a `BaseBlockRender<Block, Tile>`

The only usage of it is actually in `AEBaseBlock.getRendererInstance()` which is basically a lazy evaluated factory for a singleton. So there is nothing really breaking because it actually needs to know the `Class<?>`